test/onnx/test_utility_funs.py

```diff
@@ -731,7 +730,7 @@ def forward(self, x):
         assert next(iter).kind() == "aten::dequantize"
 
     # prim::ListConstruct is exported as onnx::SequenceConstruct for opset >= 11
-    @skipIfUnsupportedOpsetVersion([11, 12, 13])
+    @skipIfUnsupportedOpsetVersion([11, 12, 13, 14])
```

Comment:
If this test is not support opset version >= 11, it may better to use skipIfUnsupportedMaxOpsetVersion to limit scope. Then we don't need to modify in the next opset version.